scripts/smoke.js

```diff
@@ -0,0 +1,67 @@
+const smoke = require("./smoke-test.js");
+
+const tests = [
+  // {
+  //   dir: "babel",
+  //   files: "packages/babel-core/src/helpers/resolve.js",
+  //   build: "make bootstrap",
+  //   test: "make test-only",
+  //   babelOptions: {
+  //     plugins: ["syntax-flow"]
+  //   }
+  // },
+  // {
+  //   dir: "jquery",
+  //   files: "dist/jquery.js",
+  //   build: "npm run build",
+  //   test: "grunt test",
+  //   success: "872 passing",
+  // },
+  // {
+  //   dir: "html-minifier",
+  //   files: "src/htmlminifier.js",
+  //   build: "npm install && grunt dist",
+  //   test: "grunt qunit",
+  //   success: "0 failed",
+  // },
+  {
+    dir: "stylelint",
+    // files: "lib/**/*.js",
+    files: "lib/rules/declaration-block-properties-order/index.js",
```

Comment:
@boopathi I think I found one example where mangler fails. Would be great if you could look into it (and/or check with your latest/local changes to the mangler). You can see it by running `node smoke.js` after pulling git submodules (`git submodule update --init --recursive`, IIUC)